app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_likes_counter
+    like.increment!(:likes_counter)
```

Comment:
- [ ] Let's kindly update the likes counter for a **post** not for a **like**. **Hints**


`post.increment!(:likes_counter)`